### DISCUSSION TITLE
on server - nuxt instance unavailable & Fetch Error

### DISCUSSION THREAD

**Discussion by mak33v** [nuxt/nuxt#17025] [Questions]
![nuxt3](https://user-images.githubusercontent.com/31045058/185809017-05f5edd6-a9be-4560-9b4a-1caebae38518.jpg)

For about a day of work on the server in the console, here is a picture. Please tell me which way to look. At the same time, everything works.

RC8

**Comment by danielroe** [comment]
Likely you are calling a composable (like `useCookie`) outside of a setup function, middleware or plugin. That includes calling it in the catch block after an await.

**Comment by oooowl** [comment]
The same issue with me,like this:
```
// composables/useAxios.js
instance.interceptors.request.use(
  function (config) {
      const token = useCookie("token");
      config.headers.Authorization = `Bearer ${token.value}`;
})

// pages/index.vue
const { data} = await useAsyncData(() => useAxios.get("/api/xxx"));
```

**Reply by danielroe** [reply]
You should not be calling useCookie within the config function, but rather synchronously when your composable is called. It needs to run within a setup function, Nuxt plugin or Nuxt route middleware.
